pyproject.toml

```diff
@@ -1,5 +1,29 @@
 [tool.ruff]
-select = ["E", "F", "I"]
+select = [
```

Comment:
similar to [Zulip](https://github.com/zulip/zulip/blob/201debc1ea28a489b1c235d4936405b11d78d4b4/pyproject.toml#L100) and the extend-ignore comments below (line 27) can you add the reference to the annotations. 

Also for purely out of curiosity why did you use Zulip as the model and not Pandas or FastAPI?

Reply:
Added. 2 reasons: 1. because Zulip's config is more comprehensive, 2. I heard about Ruff from Zulip devs.

Reply:
Ver cool, thanks!